crates/nu-cli/Cargo.toml

```diff
@@ -76,7 +76,7 @@ rayon = "1.5.0"
 regex = "1.4.3"
 roxmltree = "0.14.0"
 rust-embed = "5.9.0"
-rustyline = { version = "8.0.0", optional = true }
+rustyline = { git = "https://github.com/kkawakam/rustyline.git", optional = true }
```

Comment:
I switched to the git version of `rustyline` because there is no release available yet that includes the merged PR kkawakam/rustyline#513